weather.noaa/resources/settings.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
would be nice if you could migrate this to the new settings format at some point. See: https://kodi.wiki/view/Add-on_settings_conversion

Automated addon: https://gitlab.com/ronie/script.settings-convert